modules/runtime/pages/performance-tuning.adoc

```diff
@@ -354,33 +353,21 @@ The following cache configurations can be defined:
 |===
 | Configuration | Purpose
 
-| connectorCacheConfig
+| connector
 | stores connector implementations for a given connector definition
 
-| processDefCacheConfig
+| processDefinition
 | stores process definition objects
 
-| userFilterCacheConfig
+| userFilter
 | stores user filter implementations for a given user filter definition
 
-| migrationPlanCacheConfig
-| not yet used
-
-| breakpointCacheConfig
-| not yet used
-
-| groovyScriptCacheConfig
+| groovyScript
 | stores compiled versions of Groovy scripts
 
-| synchroServiceCacheConfig
-| used by the benchmark test infrastructure (and has no meaning outside of it)
-
-| transientDataCacheConfig
+| transientData
 | stores transient data
 
-| platformCacheConfig
```

Comment:
no need to document how to change a cache configuration for a table containing 1 line!